custom/ilsgateway/templates/ilsgateway/base_template.html

```diff
@@ -2,6 +2,16 @@
 {% load i18n %}
 {% load hq_shared_tags %}
 
+{% block stylesheets %}{{ block.super }}
+    <link href="{% static 'nvd3/src/nv.d3.css' %}" rel="stylesheet">
+    <link href="{% static 'ilsgateway/css/ilsgateway.css' %}" rel="stylesheet">
+{% endblock %}
+
+{% block js %}{{ block.super }}
+    <script src="{% static 'd3/d3.min.js' %}"></script>
+    <script src="{% static 'nvd3/nv.d3.min.js' %}"></script>
+{% endblock %}
+
 {% block export %}
```

Comment:
i meant all the script references. ok, that sounds right, @kkrampa let us know if you disagree with this change (if those imports shouldn't be in the base template for some reason)